Lib/test/test_io.py

```diff
@@ -160,6 +160,28 @@ class PyMockRawIO(MockRawIO, pyio.RawIOBase):
     pass
 
 
+class MockRawIOWithoutFileno:
+
+    def readable(self):
+        return True
+
+    def read(self, n=None):
+        return b""
+
+    def seekable(self):
+        return True
+
+    def seek(self, pos, whence):
+        return 0   # wrong but we gotta return something
```

Comment:
Why not return `None`?

Reply:
No particular reason, I think I copied from a similar test from this file. I can give it a go with `None` as well.